packages/next-polyfill-nomodule/src/index.js

```diff
@@ -29,6 +29,7 @@ import 'core-js/features/object/get-own-property-descriptors'
 import 'core-js/features/object/keys'
 import 'core-js/features/object/is'
 import 'core-js/features/object/values'
+import 'core-js/features/object/has-own'
```

Comment:
If we want to ensure `hasOwn` is always present for [our supported browsers](https://nextjs.org/docs/architecture/supported-browsers) we'll need to add it to https://github.com/vercel/next.js/blob/canary/packages/next-polyfill-module/src/index.js as Safari 12 [supports module](https://caniuse.com/?search=type%3Dmodule) but doesn't have [`hasOwn`](https://caniuse.com/?search=hasown)

Reply:
fixed